### PR TITLE
zSystems Update

### PR DESCRIPTION
**Pull request by Loquacity**
https://github.com/SUSE/spacewalk/issues/7661

### CODE REVIEW

File: modules/installation/pages/install-ibmz.adoc [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+
+You will require an additional disk for database storage.
+This should be an [systemitem]``zFCP`` or [systemitem]``DASD`` device as these are preferred for use with [systemitem]``HYPERPAV``.
+The database storage disk should have:
+
+* At least 50{nbsp}GB for [path]``/var/lib/pgsql``
+* At least 50{nbsp}GB for each SUSE product in [path]``/var/spacewalk``
+* At least 250{nbsp}GB for each Red Hat product in [path]``/var/spacewalk``
+
+You will need to ensure you have sufficient disk storage for {productname}
```
Comment (keichwa) [inline modules/installation/pages/install-ibmz.adoc]:
```suggestion
You will need to ensure you have sufficient disk storage for {productname} before running [command]``yast2 susemanagersetup``.
```
avoid line break

File: modules/installation/pages/install-ibmz.adoc [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+before running [command]``yast2 susemanagersetup``.
```
Comment (keichwa) [inline modules/installation/pages/install-ibmz.adoc]:
```suggestion
```
cont. of the previous suggestion

File: modules/installation/pages/install-ibmz.adoc [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+By default, the {productname} file system, including the embedded database and patch directories, reside within the root directory.
+While adjustments are possible once installation is complete, it is important that specify and monitor these adjustments closely.
```
Comment (keichwa) [inline modules/installation/pages/install-ibmz.adoc]:
```suggestion
While adjustments are possible when installation is complete, it is important that you specify and monitor these adjustments closely.
```
simplify wording

File: modules/installation/pages/install-ibmz.adoc [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+For information on storage management and reclaiming disk space, see the {productname} Administration Guide.
```
Comment (keichwa) [inline modules/installation/pages/install-ibmz.adoc]: is it possible to provide a clickable link?  Wondering whether that's actually in our Admin Guide.  Maybe, it's in the SLES Admin?

File: modules/installation/pages/install-ibmz.adoc [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+* HiperSockets or Guest LAN
+* 10{nbsp}GBE, VSWITCH
+* RDMA over Converged Ethernet (RoCE)
+
+These interfaces are still included but no longer supported:
+
+* CTC or virtual CTC
+* IP network interface for IUCV
+
+The z/VM guest you want to run {productname} from will require a static IP address and hostname before you begin, as these can not easily be changed after initial installation.
```
Comment (keichwa) [inline modules/installation/pages/install-ibmz.adoc]:
```suggestion
The z/VM guest you want to run {productname} from will require a static IP address and hostname before you begin, as these cannot easily be changed after initial installation.
```
normalization

File: modules/installation/pages/install-ibmz.adoc (hunk 4 above)
Comment (Loquacity) [reply modules/installation/pages/install-ibmz.adoc]: Oh yeah, I meant to come back to that. I will need to investigate.

Comment (Loquacity) [reply modules/installation/pages/install-ibmz.adoc]: I can't see anything specific in our Admin Guide, but I think it's a good candidate for a troubleshooting  section. I'll create something in a different PR.

Comment (Loquacity) [reply modules/installation/pages/install-ibmz.adoc]: https://github.com/SUSE/doc-susemanager/pull/437 (it's a beginning)

**Comment by keichwa** [comment]
Lana Brindley <notifications@github.com> writes:

> https://github.com/SUSE/doc-susemanager/pull/437 (it's a beginning)

Thanks you!

-- 
Karl Eichwalder            SUSE Linux GmbH
PES / Documentation        Maxfeldstraße 5
                           90409 Nürnberg, Germany
GF: Felix Imendörffer, Mary Higgins, Sri Rasiah, HRB 21284 (AG Nürnberg)

**Comment by Loquacity** [comment]
Going to leave this out there until next week to give Mark Post time to review.

**Comment by Loquacity** [comment]
> Going to leave this out there until next week to give Mark Post time to review.

Looks  like no further reviews incoming. Merging.

**Pull request merged**